Use distroless image to build controller and speaker containers (#2418)
metallb/metallb: website/content/troubleshooting/_index.md

kubectl get endpointslices <my-service> -o yaml
kubectl get svc <my_service> -o yaml
```

> ### How to debug a container with distroless image

Reviewer: ```suggestion
### How to debug the speaker and the controller containers
```

Author: changed..

> With the distroless solution, if someone needs to debug inside speaker container, an ephemeral container should be used: 

Reviewer: ```suggestion
Due to the fact that both the speaker and the controller containers are based on a distroless image, an ephemeral container should be used to debug inside them: 
```

Author: modified.
